doc/gui/examples/controls/metric-layout.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+# -----------------------------------------------------------------------------------------
+# To execute this script, make sure that the taipy-gui package is installed in your
+# Python environment and run:
+#     python <script>
+# -----------------------------------------------------------------------------------------
+from taipy.gui import Gui
+
+# Layout reference can be found in the documentation: https://plotly.com/python/reference/layout/
+
+custom_layout = {
+    "width": "1000",
+    "height": "500",
+}
+
+page = """
+<|30|metric|layout={custom_layout}|>
```

Comment:
Can use width and height directly
Can you show a differentiating use case

Reply:
If you want an example of using the layout, I would want a more interesting example than just width or height. What other impact can `layout` have on a gauge?
```suggestion
<|30|metric|layout={layout}|>
```

Reply:
I've added the background color, margin & font size for the example